feedingwebapp/src/Pages/Settings/CustomizeConfiguration.jsx

```diff
@@ -337,17 +428,19 @@ const CustomizeConfiguration = (props) => {
     }
   }, [localCurrAndNextMealState, props.webrtcURL, robotMotionProps, faceDetectedCallback])
 
+  const resetToPresetSuccessCallback = useRef(() => moveToButtonClicked(props.startingMealState))
```

Comment:
TODO: I have to add a test case for this to verify it works, because wrapping a `useCallback` within a `useRef` seems a bit sus.